recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/defconfig

```diff
@@ -4588,7 +4592,23 @@ CONFIG_ARCH_TEGRA_210_SOC=y
 # CONFIG_SOC_TI is not set
 CONFIG_RCAR_THERMAL_EMS_ENABLED=y
 CONFIG_RCAR_DDR_BACKUP=y
-# CONFIG_PM_DEVFREQ is not set
+CONFIG_PM_DEVFREQ=y
```

Comment:
What have changed in the system so that you have CONFIG_PM_DEVFREQ enabled?

Reply:
See my comment above. I don't know what the base defconfig came from. The changes you are facing are changes between defconfig which @iartemenko decided to put here as "some base" and defconfig I pulled in from bare yocto and just changed some necessary configs, mostly Xen related.